cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -150,6 +150,11 @@ spec:
                   type: boolean
             placement: {}
             resources: {}
+            cleanupPolicy:
+              properties:
+                deleteDataDirOnHosts:
+                  type: string
+                  pattern: ^$|^yes-really-destroy-data$
```

Comment:
Why not a bool? Just to make sure this was **really** desired and thus more painful than writing `true`?

Reply:
Yes, exactly. We want to be really-really sure they want to destroy the data :)